target/linux/mpc85xx/files/arch/powerpc/boot/dts/firebox-t10.dts

```diff
@@ -103,6 +103,24 @@
 						reg = <0xc0000 0x40000>;
 						label = "device_id";
 						read-only;
+
+						nvmem-layout {
+							compatible = "fixed-layout";
+							#address-cells = <1>;
+							#size-cells = <1>;
+
+							macaddr_device_id_1830: mac-address-hex@1830 {
+								reg = <0x1830 0xc>;
+							};
+
+							macaddr_device_id_1844: mac-address-hex@1844 {
+								reg = <0x1844 0xc>;
+							};
+
+							macaddr_device_id_1858: mac-address-hex@1858 {
+								reg = <0x1858 0xc>;
```

Comment:
This is wrong... @neheb since I would prefer to fix this instead of revert, do you have entry of what is actually set in this device? Is it really 0xc? aka mac in the form of text 000000000000 or with : 

Do you have reference?

Reply:
I have no idea. This commit was modeled after https://github.com/openwrt/openwrt/pull/13997

Probably better to revert as I do not have the hardware.

Reply:
sad.... do you know by chance anyone that have this device? @blocktrron ?

Reply:
it uses colons.